app/assets/javascripts/admin/views/editions/_form.js

```diff
@@ -265,6 +266,27 @@
         $(this).parent().hide().next().removeClass('if-js-hide');
         e.preventDefault();
       });
+    },
+
+    showImageUploaderIfCustomImage: function showImageUploaderIfCustomImage() {
+      var $imageUploader = $('.image_section .js-show-image-uploader');
+      var $customImage = $('#edition_image_display_option_custom_image');
```

Comment:
I'd rename this to `$customImageRadioButton` so it's a little clearer what it's referring to.